### PR TITLE
Add links to view doc for scanners ui

### PR DESCRIPTION
**Pull request by AllForNothing**
New added parts:
image
image
image
UX:https://www.figma.com/file/0bSkyzvchcOCrwSWROIivtIb/Harbor-v1.9?node-id=1426%3A43663
Env.:https://10.161.67.1:8090

Signed-off-by: sshijun <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/base/harbor-shell/harbor-shell.component.ts [hunk 1]
```diff
@@ -82,8 +88,29 @@ export class HarborShellComponent implements OnInit, OnDestroy {
         this.searchCloseSub = this.searchTrigger.searchCloseChan$.subscribe(close => {
             this.isSearchResultsOpened = false;
         });
+        if (!(localStorage && localStorage.getItem(HAS_SHOWED_SCANNER_INFO) === 'yes')) {
```
Comment (jwangyangls) [inline src/portal/src/app/base/harbor-shell/harbor-shell.component.ts]: ‘yes’?

File: src/portal/src/app/base/harbor-shell/harbor-shell.component.ts [hunk 2]
```diff
@@ -82,8 +88,29 @@ export class HarborShellComponent implements OnInit, OnDestroy {
...
+        if (localStorage) {
+            localStorage.setItem(HAS_SHOWED_SCANNER_INFO, 'yes');
+        }
+        this.showScannerInfo = false;
     }
 
+    getDefaultScanner() {
+        this.scannerService.getScanners()
+            .subscribe(scanners => {
+                if (scanners && scanners.length) {
```
Comment (jwangyangls) [inline src/portal/src/app/base/harbor-shell/harbor-shell.component.ts]: this.showScannerInfo = scanners.some(scanner =>scanner.is_default);   
One line of code can do it

File: src/portal/src/app/base/harbor-shell/harbor-shell.component.ts (hunk 1 above)
Comment (jwangyangls) [reply src/portal/src/app/base/harbor-shell/harbor-shell.component.ts]: hardcode

File: src/portal/src/app/base/harbor-shell/harbor-shell.component.ts (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/app/base/harbor-shell/harbor-shell.component.ts]: fixed

File: src/portal/src/app/base/harbor-shell/harbor-shell.component.ts (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/base/harbor-shell/harbor-shell.component.ts]: fixed

**Comment by xaleeks** [comment]
lgtm, but we really should have @lweitzman review all these before being merged.

**Pull request merged**